php/class-photonfill.php

```diff
@@ -855,6 +855,16 @@ public function get_image_classes( $class = array(), $attachment_id = null, $siz
 		 * @return string Image markup.
 		 */
 		public function get_picturefill_html( $html, $post_id, $post_thumbnail_id, $size, $attr ) {
+			// Core permits $attr to be a query string or an array, but Photonfill expects an array.
```

Comment:
I like this, a lot.  How about also updating the param phpdoc to reflect this change?